### PR TITLE
Fix cookie consent on videos

### PR DESCRIPTION
**Pull request by katjuell**

### CODE REVIEW

**Review by bretthayes — DISMISSED** [review]

File: blogposts/2022/sourcegraph-extension-for-vs-code.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ gradle org\.apache\.logging\.log4j ['"] 2\.(0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|1
 As you can see in the video below, the global search returns several repositories that require further investigation. I finally changed the search context to Sourcegraph to narrow down my search scope to find all the repositories affecting our organization.
 
 <video loop autoplay muted playsinline>
-  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.webm" type="video/webm" />
-  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.mp4" type="video/mp4" />
+  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.webm" type="video/webm" data-cookieconsent="ignore" />
+  <source src="https://storage.cloud.google.com/sourcegraph-assets/blog/vs-code-extension/vscode-log4j.mp4" type="video/mp4" data-cookieconsent="ignore" />
```
Comment (bretthayes) [inline blogposts/2022/sourcegraph-extension-for-vs-code.md:52]: The only one that doesn't work is this one which we are already aware of

**Review by bretthayes — APPROVED** [review]
Looks good! Tested each page in a new incognito window and the assets load. Still unsure why that one video in my comment below doesn't load though.

File: blogposts/2022/sourcegraph-extension-for-vs-code.md (hunk 1 above)
Comment (katjuell) [reply blogposts/2022/sourcegraph-extension-for-vs-code.md:52]: Yeah, let's circle back and try to get to the bottom of that as a follow up!

File: blogposts/2020/announcing-sourcegraph-3.16.md [hunk 2]
```diff
@@ -48,15 +48,15 @@ Sourcegraph couldn't be what it is without the community.
 Go in-depth with the 3.16 release by watching the recorded livestream with Product Manager [Christina Forney](https://twitter.com/christina4nee) and Software Engineers [Thorsten Ball](https://twitter.com/thorstenball), [Loïc Guychard](https://github.com/lguychard), and [Eric Fritz](https://github.com/efritz).
 
 <div class="container my-4 video-embed embed-responsive embed-responsive-16by9 ">
-    <iframe class="embed-responsive-item" src="https://www.youtube-nocookie.com/embed/TgbiCEGaFyU?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0" allowfullscreen="" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" frameborder="0"></iframe>
+    <iframe class="embed-responsive-item" data-cookieconsent="ignore" src="https://www.youtube-nocookie.com/embed/TgbiCEGaFyU?autoplay=0&amp;cc_load_policy=0&amp;start=0&amp;end=0&amp;loop=0&amp;controls=1&amp;modestbranding=0&amp;rel=0" allowfullscreen="" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" frameborder="0"></iframe>
```
[the remark this answers is not in the document]
Comment (katjuell) [reply blogposts/2020/announcing-sourcegraph-3.16.md:51]: Gotcha! @bretthayes and I were seeing some behavior with these assets in incognito mode that this change seemed to resolve, but happy to revert if this is likely to cause problems.

[the remark this answers is not in the document]
Comment (katjuell) [reply blogposts/2020/announcing-sourcegraph-3.16.md:51]: Just went back to check on this in an older deploy preview and it does seem to be an issue in incognito (that seems to be resolved now in prod). Looking at that console error, though, and I'm wondering if it's a question of toggling something so we can re-enable our original settings for Youtube videos that are iframed.

**Update**: It looks like the `id` in that console message matches what we have in our [codebase](https://github.com/sourcegraph/about/blob/d34577031bf0a6ecffc9cc757c1754602c3d456b/website/src/html.tsx#L87), but **not** what we have in Cookiebot. I wonder if this is just a question of updating our Cookiebot id? Not sure when or how it got changed. Looks like the original addition to this codebase was [back in Feb 2021](https://github.com/sourcegraph/about/commit/d96a0cdb99ca30419f7f338d3732de28a91aead4)

https://user-images.githubusercontent.com/43454804/157893584-15616d3d-e233-44db-943b-9190574763f5.mov

[the remark this answers is not in the document]
Comment (katjuell) [reply blogposts/2020/announcing-sourcegraph-3.16.md:51]: I'm going to re-enable this for iframed youtube videos, and add the deploy domain to the list of acceptable aliases in Cookiebot and see how these look

**Pull request merged**